plugins/blip/basics/components/sitechange/Selector.js

```diff
@@ -123,7 +123,7 @@ var Selector = React.createClass({
   renderOptions: function() {
     var self = this;
 
-    if (!self.props.selectorMetaData.hasOwnProperty('canUpdateSettings')) {
+    if (!self.props.selectorMetaData.hasOwnProperty('canUpdateSettings') || !self.props.selectorMetaData.canUpdateSettings) {
```

Comment:
Well not quite. You're checking to see if the property _doesn't_ exist or if it's falsey. Are there definitely cases where the property won't exist, or is the check just defensive? `!condition || !condition` is always hard to mentally compute.

Yeah, the more I read this the more it seems that the falsiness check should suffice, or am I missing something?